src/store/modules/auth.store.js

```diff
@@ -50,8 +50,26 @@ export const authStore = {
                         dispatch('alertStore/error', error, { root: true });
                     }
                 );
-        }     
+        },
+            
+        resetPassword({ dispatch, commit },username, password) {
+            authService.resetPassword(username, password)
+            .then(
+                user => {
+                    let storedProfile = JSON.parse(localStorage.getItem('user'));
+                    storedProfile['profile'] = user.profile;
+                    localStorage.setItem('user', JSON.stringify(storedProfile));
```

Comment:
You are already overwriting `user` from service function. Why are you again updating the user ? Password change does not require user to be updated in the frontend. @vikas339